docs/tutorials/performance.rst

```diff
@@ -4,7 +4,7 @@ Performance
 LSDB is a high-performance package built to support the analysis of large-scale astronomical datasets.
 One of the performance goals of LSDB is to add as little overhead over the input-output operations as possible.
 We achieve this aim for catalog cross-matching, spatial and data filtering operations by using
-the `HiPSCat <https://github.com/astronomy-commons/hipscat>`_ data format,
+the `HATS <https://github.com/astronomy-commons/hats>`_ data format,
```

Comment:
Unless I miss my guess, this is assuming that the repo name has changed, doesn't it?

Reply:
Yeah - it is. This will be a 404 for another week. We'll need to do a full scan of our documentation once everything has moved to update all references. But sphinx doesn't check that links aren't dead, so CI will still be ok.